mne/io/edf/edf.py

```diff
@@ -204,8 +204,8 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                     # make sure events without duration get one sample
                     n_stop = n_stop if n_stop > n_start else n_start + 1
                     if any(stim[n_start:n_stop]):
-                        raise NotImplementedError('EDF+ with overlapping '
-                                                  'events not supported.')
+                        warn('EDF+ with overlapping events'
+                             ' are not fully supported')
```

Comment:
We had some discussion of this before regarding potentially adding the event numbers instead of just overwriting them. Not sure what the best policy is.